packages/gen-ai-hub/src/utils/deployment-cache.ts

```diff
@@ -0,0 +1,83 @@
+// TODO: should we move this to @sap-cloud-sdk/connectivity? or add some kind of exception to the rule if possible?
```

Comment:
@deekshas8 @tomfrenken Did we align on whether the SAP Cloud SDK for AI is "allowed" to reach to the internal API? I think it should be ok, but we should then fix the lint rule for this.

Reply:
I agree, however, only for internals of the sap cloud SDK, not internals of the AI SDK, which mich make the rule a little more complicated.

Reply:
Seems like Junjie did: https://github.tools.sap/AI/gen-ai-hub-sdk-js-backlog/issues/40